tiledb/sm/query/deletes_and_updates/deletes_and_updates.cc

```diff
@@ -128,7 +128,7 @@ Status DeletesAndUpdates::dowork() {
 
   // Check that the update values are valid.
   for (auto& update_value : update_values_) {
-    std::ignore = update_value.check(array_schema_);
```

Comment:
UpdateValue::check could be converted to a void function that throws exceptions if you're up for a very small refactor.